packages/round-manager/src/features/api/programService.ts

```diff
@@ -0,0 +1,124 @@
+// import { ethers } from "ethers"
+import { api } from "."
+import { global } from "../../global"
+
+
+export interface Program {
+  /**
+   * The on-chain unique program ID
+   */
+  id?: string;
+  /**
+   * The name of the program, this can be update
+   */
+  name: string;
+  /**
+   * Addresses of wallets that will have admin privileges to operate the Grant program
+   */
+  operatorWallets: Array<string>;
+}
+
+function sleep(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+/** Contract ABI for the ProgramFactory in the Human-Readable format */
+const CONTRACT_ABI = [
```

Comment:
could we throw the contract ABI into it's own folder and import them ? 
that way you don't have to touch the service for ABI upgrades.

Reply:
done ✅